src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -720,13 +749,50 @@ function MoneyRequestConfirmationList({
                     style={[styles.moneyRequestMenuItem]}
                     titleStyle={styles.flex1}
                     onPress={() => Navigation.navigate(ROUTES.MONEY_REQUEST_STEP_DISTANCE.getRoute(action, iouType, transactionID, reportID, Navigation.getActiveRouteWithoutParams()))}
+                    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
```

Comment:
NAB: It seems like we shouldn't need this comment

Reply:
This and other follow ups are handled here https://github.com/Expensify/App/pull/40711#top